website/other/bezier-rs-demos/src/features/subpath-features.ts

```diff
@@ -40,32 +40,52 @@ const subpathFeatures = {
 		sliderOptions: [tSliderOptions],
 		chooseTVariant: true,
 	},
-	"intersect-linear": {
+	"Intersect (Line Segment)": {
```

Comment:
The shorthand name of the feature should not be modified (here, and for the other intersect features)